tests/tensor_search/test_api_utils.py

```diff
@@ -113,6 +113,31 @@ def test_add_docs_params_orchestrator(self):
         assert result.docs == [{"test": "doc"}]
         assert result.image_download_headers == {"header1": "value1"}
 
+    def test_add_docs_params_orchestrator_defaults(self):
+        """
+        Ensures that certain defaults are correct when not specified upon calls to 
+        add_docs_params_orchestrator (like use_existing_tensors, auto_refresh, etc)
+        """
```

Comment:
thanks for proactively filling gaps in our current set of tests

Reply:
no worries! this will be a good template to build upon when future parameters/features are implemented.